### PR TITLE
Support more than one token per node and params per token

### PR DESCRIPTION
**Pull request by yaslama**

### CODE REVIEW

**Review by JssDWt — COMMENTED** [review]
Looking good! This will work.
2 comments about backward compatibility.

File: postgresql/intercept_store.go [hunk 1]
```diff
@@ -15,6 +15,11 @@ import (
 	"github.com/jackc/pgx/v4/pgxpool"
 )
 
+type extendedParams struct {
+	Token  string                       `json:"token"`
+	Params interceptor.OpeningFeeParams `json:"fees_params"`
```
[the remark this answers is not in the document]
Comment (yaslama) [reply postgresql/intercept_store.go:20]: The json is stored in the table only if there is a params

File: postgresql/intercept_store.go [hunk 2]
```diff
@@ -51,15 +56,15 @@ func (s *PostgresInterceptStore) PaymentInfo(htlcPaymentHash []byte) (*intercept
 		}
 	}
 
-	var params *interceptor.OpeningFeeParams
+	var extParams *extendedParams
 	if p != nil {
-		err = json.Unmarshal([]byte(*p), &params)
+		err = json.Unmarshal([]byte(*p), &extParams)
```
[the remark this answers is not in the document]
Comment (yaslama) [reply postgresql/intercept_store.go:61]: Yes. But it cannot be done in a regular migration but can be done between the stop and start of lspd.

[the remark this answers is not in the document]
Comment (JssDWt) [reply postgresql/intercept_store.go:61]: Yes exactly if you stop lspd, run a migration that wraps the params and then start lspd with the new code that should work.

**Pull request merged**